sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -547,14 +545,14 @@ def add_participants(
     @distributed_trace
     def remove_participant(
         self,
-        user,  # type: CommunicationUserIdentifier
+        user,  # type: CommunicationIdentifier
```

Comment:
In this case we eventually want to support a union of type here - either an Identifier, or a ChatParticipant.

I would prefer that we go with `participant`, but then I would also need to add the support for taking `ChatParticipant` as input here. Which I'm happy to do - because this is what I wanted.
But would need your teams approval ;)

Reply:
I would prefer if we don't diverge from the interface we agreed upon across sdks. It feels also like creating 2 nested objects just to be able to remove the user is a bit cumbersome.

Reply:
@juancamilor So you agree that we should rename to `identifier` here? That's what the APIViews you linked to are doing (correctly).